File: Python/7 KYU/Selective fear of numbers.py
def am_I_afraid(day, num):
    if (day == 'Monday' and num == 12) or (day == 'Tuesday' and num > 95) or (day == 'Thursday' and num == 0) or (
            day == 'Wednesday' and num == 34) or (day == 'Friday' and num % 2 == 0) or (
            day == 'Saturday' and num == 56) or (day == 'Sunday' and abs(num) == 666):
        return True
    else:
        return False

File: Ruby/7 KYU/Selective fear of numbers.rb
def am_I_afraid(day,num)
  if (day == 'Monday' && num == 12) || (day == 'Tuesday' && num > 95) || (day == 'Thursday' && num == 0) || (day == 'Wednesday' && num == 34) || (day == 'Friday' && num % 2 == 0) || (day == 'Saturday' && num == 56) || (day == 'Sunday' && num.abs == 666)
    true
  else
    false
  end
end
